### PR TITLE
Add man page for renderd.conf

### LINKED ISSUE #199: Missing man page for renderd.conf
The man page for renderd mentions a man page for renderd.conf (see https://github.com/openstreetmap/mod_tile/blob/master/docs/renderd.8#L41), which is not installed.

The man page for renderd.conf should be generated and installed.

### PR DESCRIPTION
**Pull request by hummeltech**
Resolves #199

### CODE REVIEW

File: docs/man/renderd.conf.5 [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+.PP
+.B renderd.conf
+is an INI configuration file for use with
+.B renderd(1).
+.PP
+The default location is
+.B '/etc/renderd.conf'
+(e.g.
+.B RENDERD_CONFIG)
+.PP
```
Comment (Faerbit) [inline docs/man/renderd.conf.5:23]: I think the closing parens shouldn't be bolded:
```suggestion
.B RENDERD_CONFIG
)
.PP
```

The same thing present multiple times

**Comment by Faerbit** [comment]
I'm not familiar with renderd or OSM tooling in general, but I have a couple of remarks:
* I am especially not familiar with your build infrastructure, so I cannot comment on that
* Some options could use a little bit more explanation (e.g. "htcphost Only used by renderd."). But I guess you are aware of this, and do not have any detail knowledge about them either
* I am not a native english speaker, but I think you are using "e.g." wrong. I don't know what you mean, when you say: 
  "The  default value is '/var/run/renderd/renderd.sock' (e.g.  RENDERD_SOCKET)"

But all in all, it looks pretty good :) Good job :+1:

**Comment by hummeltech** [comment]
Thank you for the comments, yes I do believe in many cases I am indeed improperly using `e.g.`, especially where it applies to the `macro definitions` as you pointed out, I will address those.

File: docs/man/renderd.conf.5 (hunk 1 above)
Comment (hummeltech) [reply docs/man/renderd.conf.5:23]: Yes, I tried that and it adds a tab after the bolded term. I will try and use `\fB`...`\fR`

**Pull request merged**